CONTRIBUTING.md

```diff
@@ -315,9 +315,18 @@ even patch releases may contain [non-backwards-compatible changes](https://semve
 
 ### Creating a new release
 
-1. Update the version with `rg 0.0.269 --files-with-matches | xargs sed -i 's/0.0.269/0.0.270/g'`
-1. Update `BREAKING_CHANGES.md`
-1. Create a PR with the version and `BREAKING_CHANGES.md` updated
+We use an experimental in-house tool for managing releases.
+
+1. Install `rooster`: `pip install git+https://github.com/zanieb/rooster@main`
```

Comment:
Does the `@main` have an effect here?

Reply:
It shouldn't, but until it's on PyPI I might tag versions and pin them as changes are made going forward.

We could drop it